docs/02-app/01-building-your-application/02-data-fetching/01-fetching.mdx

```diff
@@ -131,7 +131,7 @@ export default async function Page() {
 
 If you are not using any [dynamic functions](/docs/app/building-your-application/rendering/server-components#dynamic-rendering) anywhere else in this route, it will be prerendered during `next build` to a static page. The data can then be updated using [Incremental Static Regeneration](/docs/app/building-your-application/data-fetching/incremental-static-regeneration).
 
-If you do _not_ want to cache the response from the database, you can add the following to your file:
+If you want to _avoid_ caching the database response in the statically rendered page, you can add the following to your file:
```

Comment:
Database results are included in the rendered page and cached as a static page. Since full-route caching is used, the client will not receive updated data from the database from this route. In this case, the database response itself isn’t cached, but the route that relies on the data is. Just for clarification.